base.mk

```diff
@@ -50,7 +51,17 @@ PRODUCT_COPY_FILES += \
 
 # LineageOS build may need this to make NFC work
 PRODUCT_PACKAGES += \
-        NfcNci  
+        NfcNci \
+	libnfc-nci \
+    	Tag \
+    	com.android.nfc_extras \
+    	nfc_nci.pn54x.default
```

Comment:
Ok, so to be clear, there is a android.hardware.nfc@1.0-service running in /vendor (?) loading /system/lib/hw/nfc_nci.pn54x.default.so? And running binderized? Or is this only a passthrough HAL?

Considering that there is no reason for AOSP to keep such compatibility, I think this should perhaps go to vndk to keep compatibility. If the HAL is passthrough, I guess we'll also need to build our own treblized HAL. Not too big deal, but still a bit annoying